### ISSUE TITLE
Convert [$domain=/regex/] to $domain=/regex/ for uBO version of lists

### ISSUE THREAD

**Issue by Yuki2718** [AdguardTeam/FiltersCompiler#204]
https://github.com/gorhill/uBlock/commit/b1de8d3fe48755da58268ba86dccd5d76940b613

**Comment by punisher018** [comment]
Can you have a look at the filters of the dizipal811.com topic above? Their format is not supported by ublock.

**Comment by Yuki2718** [comment]
I missed the notification, only noticed when I'm checking email. But what to look for? I don't see any regex filter for dizipal* in uBO version of AdGuard Turkish filter.

**Comment by punisher018** [comment]
@Yuki2718, @Alex-302 added the filters suitable for uBO. So there is no problem for now. (If the number of domain addresses of the site exceeds 825, it will be necessary to update.) But the regex filters are still there, if you want to take a look. (Line 5510 - 5514)
`https://github.com/AdguardTeam/AdguardFilters/commit/cbe528924cf2a93f753bb115467d7e04119ceee0`

**Comment by Yuki2718** [comment]
yeah, I found and don't know why missed them. But as said, `[$domain=/dizipal\d+.com/]` is not supported by uBO.
